sig-aws/README.md

```diff
@@ -13,7 +13,7 @@ Covers maintaining, supporting, and using Kubernetes hosted on AWS Cloud.
 The [charter](charter.md) defines the scope and governance of the AWS Special Interest Group.
 
 ## Meetings
-* Regular SIG Meeting: [Fridays at 9:00 PT (Pacific Time)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
+* Regular SIG Meeting: [Fridays at 9:00 PT (Pacific Time)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) (biweekly starting Friday January 11, 2019). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
```

Comment:
@justaugustus @spiffxp  - what is the value of this link? (https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit) seems broken for most SIGs. Do we even need this?

Reply:
@d-nishi -- there was a saga around bad actors on Zoom:
- https://groups.google.com/d/topic/kubernetes-dev/sHkr7w9n1yc/discussion
- https://groups.google.com/d/topic/kubernetes-sig-leads/2cieLWxBqeM/discussion

which led to updates in https://github.com/kubernetes/community/blob/master/communication/zoom-guidelines.md

I believe these were put in as a placeholder until each SIG is under moderation, but @parispittman + @castrojo would have more context.

Reply:
Thanks @justaugustus 

@parispittman @castrojo -- what action is pending for us to make this link meaningful?

Reply:
I'm going to defer to them, but at a minimum I would expect the meeting needs to be setup per https://github.com/kubernetes/community/blob/master/communication/zoom-guidelines.md#setting-up-your-meeting-and-moderation

I say this as someone who still needs to do the same for sig-testing's meetings 😭